project/proposals/119-custom-connectors.md

```diff
@@ -67,28 +67,38 @@ with some properties omitted:
     "name": "Swagger API Connector",
     "description": "Swagger API Connector",
     "icon": "fa-link",
+    "camelConnectorGAV": "io.syndesis:rest-swagger-connector:@syndesis.version@",
+    "camelConnectorPrefix": "swagger-operation",
+    "connectorGroup" : {
+      "id": "swagger-connector-template" // inherited by all custom connectors generated from this template as `connectorGroup` and `connectorGroupId`
```

Comment:
JSON does not allow comments

Reply:
Yeah, I have no idea how to explain these properties other than to provide inline comments. And I'm too lazy to convert to asciidoc to have callouts.